### DISCUSSION TITLE
Recommended system سیستم پیشنهادی

### DISCUSSION THREAD

**Discussion by akv79** [hiddify/Hiddify-Manager#167] [Q&A سوالات رایج] [answered]
سلام خسته نباشید
سیستم پیشنهادیتون برای اجرای پنل چیه؟
مورد دوم اینکه سرور مجازی کلاد رو توصیه میکنید یا همون سرور های معمولی؟
سوال آخر هم اینکه خرید سرور از سایت های ایرانی میتونه خطری داشته باشه یا نه؟
با تشکر

**Comment by arman2012** [comment, marked as answer]
سیستم پیشنهادی حداقل رم یک گیگ سی پی یو یک هسته. 
 بسته به تعداد اشخاص متصل باید قوی تر باشه
 سرور ها مدلش فرقی نداره اوبونتو باشه
 بله سرور از سایتای ایرانی مصرف بره بالا میبندن

**Comment by akv79** [comment]
رم 2 تا چند تا یوزر رو جواب میده؟
سرور از سایت های ایرانی اگه بگیریم امکانش هست دچار پیگرد قانونی قرار بگیریم یا که نهایتش سرور فیلتر یا بسته میشه؟

**Reply by arman2012** [reply]
رم مهم نیست سی پی یو محدودت میکنه رم برای 400 نفر فوقش 1 گیگ ببره
متوجه حرف بنده نشدی. فروش فیلتر شکن جرمه به طبع میتونی مورد پیگرد قرار بگیری

**Comment by akv79** [comment]
ببخشید که خیلی سوال میپرسم چون میخوام تا جایی که میشه هزینه ها رو کم کنم
به نظر شما دو هسته سی پی یو تا چند تا یوزر رو جواب میده؟
راجع به قضیه پیگرد هم من قصد فروش نداشتم. میخوام برای اعضای خانواده راه اندازی کنم فقط نگران بودم که ممکنه اگه سرور از سایت های داخلی بگیرم و به هر دلیلی متوجه بشن که فیلترشکن راه انداختم مورد پیگرد قانونی قرار بگیرم چون بالاخره تمام مشخصات رو دارن یا اینکه فقط به بستن سرور قناعت میکنن.
با تشکر

**Comment by Behzad9** [comment]
سلام
با رم 512 قابل نصب و اجرا نیست؟
